### DISCUSSION TITLE
Nextcloud AIO 6.1.1 - Cannot print documents - Collabora

### DISCUSSION THREAD

**Discussion by elephantastyczny** [nextcloud/all-in-one#2863] [Questions] [answered]
Hi All,

After switching from 6.0.0 to 6.1.1 I can't print any of the document that I've on Nextcloud.

I believe those 2 events are corresponding with this issue.
Deprecated event type for OCP\IPreview:PreviewRequested: Symfony\Component\EventDispatcher\GenericEvent is used

and

 GuzzleHttp\Exception\ClientException: Client error: `POST https://FQDN//lool/convert-to/png` resulted in a `404 Not Found` response: <!DOCTYPE html> <html class="ng-csp" data-placeholder-focus="false" lang="pl" data-locale="pl_PL" translate="no" > <hea (truncated...)

Running behind Caddy Reverse Proxy
Docs are opening fine

Nextcloud container logs:
System config value enabledPreviewProviders => 0 set to string OC\Preview\Imaginary
System config value preview_imaginary_url set to string http://nextcloud-aio-imaginary:9000
Imaginary container running fine

![image](https://github.com/nextcloud/all-in-one/assets/120596179/eceb583e-9550-497c-aad1-4e77ffc39d90)

Any suggestions?

**Comment by szaimen** [comment, marked as answer]
See https://github.com/CollaboraOnline/online/issues/6624

**Comment by elephantastyczny** [comment]
But is it doable in AIO?

"
First create a backup.
Deactivated and removed the Collabora Online - Built-in CODE Server 23.5.5 in Nextcloud under Apps (via browser).
Downloaded and unpacked the following version (via terminal): [https://github.com/CollaboraOnline/richdocumentscode/releases/download/22.5.1301/richdocumentscode.tar.gz](https://github.com/CollaboraOnline/online/issues/url)
Moved the unpacked directory to .../nextcloud/apps and set the permissions (under debian for www-data)
Nextcloud opened again in the browser: the installer for the version just downloaded will start there
Now version 22.5.1301 is installed and I ignore the update notice to 23.5.5"

**Reply by szaimen** [reply]
No, it is not doable. You need to wait for collabora to fix it. Then we can update collabora here.

**Reply by elephantastyczny** [reply]
Ok, thanks!
